pkg/proto/types.go

```diff
@@ -1724,6 +1739,10 @@ func (o *OrderV4) GetExpiration() uint64 {
 	return o.Expiration
 }
 
+func (o *OrderV4) GetAttachment() Attachment {
+	return o.Attachment
+}
```

Comment:
And btw, attachment field should be also included in `EthereumOrderV4` signature. Check [this](https://github.com/wavesplatform/Waves/pull/3851/files#diff-05f36090be8ff4d1b78fc888ad89c2221799611fcb7b40f273d174d856ff766dR48)